### ISSUE TITLE
[OS X] Fresh install, unable to login

### ISSUE THREAD

**Issue by deiga** [keybase/client#3544]
My app just keeps spinning on the username screen and nothing happens. Restarting doesn't have any effect

![screen shot 2016-07-20 at 16 57 27](https://cloud.githubusercontent.com/assets/27202/16989004/14ff340e-4e9b-11e6-8db3-d9546cb2fc3e.png)

Keybase GUI Version: 1.0.16-20160715110019+f0caa34

**Comment by cjb** [comment]
Hi, sorry about this, please run "keybase log send".  Thanks!

**Comment by deiga** [comment]
@cjb I'd love to, but my CLI can't find the command `keybase`. Where is the executable installed?ke

**Comment by cjb** [comment]
Huh!  Are you on the new macOS Sierra beta?

**Comment by deiga** [comment]
Nope, El Capitan 10.11.5

**Comment by deiga** [comment]
@cjb I ran `Keybase.app/Contents/MacOS/Keybase` from the terminal and it spat out a bunch of logs, here's an excerpt https://gist.github.com/deiga/eb3301d7c2f67328d29bc3b090e8a669

**Comment by deiga** [comment]
I also found the executable and ran `keybase log send`, my log ID is 87c47bc4abce3c54aa69541c

**Comment by iFreilicht** [comment]
I have the same issue under Windows 10, here's my log ID: 5c6e9448de9449c02a401b1c

**Comment by deiga** [comment]
I'd love to solve this in any other way than rebooting my machine. Or at least get a fix, so that future installations don't need a reboot.

This seems like a simple file creation/permission problem, so it should be solvable without booting the host machine

**Comment by eswenson1** [comment]
I have the same issue on Mac OS X 10.11.5 (El Capitan!). I cannot login via the Keybase client application.

my log ID:  8f7ea5394ffa194e045a301c

**Comment by gabriel** [comment]
Yeah there is a known issue on OS X where the installer is getting stuck and a reboot fixes. (In the GUI log you'll see the installer get stuck and never complete). 

It seems like it's not a random issue though... I'll take a look.

**Comment by gabriel** [comment]
BTW, haven't forgot about this issue, we're working on some installer enhancements which will help us diagnose this.

**Comment by gabriel** [comment]
If you are still having this problem, here is a version with some enhancements around install:
https://prerelease.keybase.io/darwin/Keybase-1.0.17-20160817191833%2Be83e8a5.dmg
